definitions/divorce/json/ComplexTypes/ComplexTypes.json

```diff
@@ -203,6 +203,55 @@
     "ElementLabel": "File name",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ConfidentialDivorceDocument",
+    "ListElementCode": "DocumentType",
```

Comment:
I don't know why it's here. It was a part of:
https://github.com/hmcts/div-ccd-definitions/pull/423

already merged into master. Should not be displayed here.